<launch>
  <!-- <arg name="google_application_credentials" default="$(env HOME)/Acciona-Pepper-4deddda971c5.json"/>-->
  <arg name="google_application_credentials" default="$(env HOME)/df_api.json"/>
  <arg name="config_file" default="$(find dialogflow_ros)/config/params.yaml"/>

  <rosparam command="load" file="$(arg config_file)" ns="/dialogflow_client"/>
  
  <node name="dialogflow_client" pkg="dialogflow_ros" type="dialogflow_client.py" output="screen">
     <param name="google_application_credentials" value="$(arg google_application_credentials)"/>
  </node>
</launch>
